README.md

```diff
@@ -2,91 +2,58 @@
 
 # Installation Guide
 
-## Build and Run
-
-### Step 1. Clone the github repo
+## Step 1: Clone the github repo
 ```sh
 git clone https://github.com/Suwi-inc/LBS-server.git
 cd LBS-server
 ```
 
-### Step 2. Install [Docker](https://docs.docker.com/engine/install/)
-
-### Step 3: Create a .env file in the root of the project
-
-```.env
-POSTGRES_USER={username}
-POSTGRES_PASSWORD={password}
-POSTGRES_DB={database}
-SECRET_KEY={string}
-REMOTE_IP={host}
-REMOTE_DB_PORT={port}
-DB_URI=postgresql+psycopg2://{username}:{password}@{host}:{port}/{database}
+## Step 2: Create a .env file in the root of the project and decide which option is more appropriate
+There are 2 entites that the product consists of: python web API and postgreSQL DBMS. 
+There are also 3 options to set the environment up:
+1. Using docker for both entities
+2. Using docker for web API, db is hosted somewhere else
+3. No docker, python server and postgres are installed locally
+
+The env file:
+```dotenv
+# for all cases
+DATABASE_USER={the user of the db}
+DATABASE_PASSWORD={the user's password}
+DATABASE_LBS_NAME={main database name}
```

Comment:
The names of the databases are actually fixed in DataBaseSchema.SQL file. They are `lbs_data` and `lbs_logs`. Maybe, use them as constants in the server implementation itself?